### ISSUE TITLE
Crossplane CLI on windows has no beta commands

### ISSUE THREAD

**Issue by markphillips100** [crossplane/crossplane#5476]
### What happened?
Crossplane CLI on windows has no beta commands.

### How can we reproduce it?
1. Using a gitbash terminal on windows, install latest Crossplane CLI tool.
2. Run `crossplane -h` to see commands.  Note the following commands listed:
```
Commands:
  core start
    Start Crossplane controllers.

  core init
    Make cluster ready for Crossplane controllers.

  rbac start
    Start Crossplane RBAC controllers.

  rbac init
    Initialize RBAC Manager.
```

### What environment did it happen in?
$ crossplane -v
v1.15.0

**Comment by negz** [comment]
@markphillips100 You want `crank.exe`. The binary you ran with `core start` etc is the Crossplane controller manager, not the CLI.

For historical reasons, we build the Crossplane controller manager as `crossplane` (`crossplane.exe` on Windows). The CLI is built as `crank.exe`.

Did you install using https://github.com/crossplane/crossplane/blob/master/install.sh? It should take care of downloading `crank.exe` and renaming it, but I doubt the Windows codepath gets exercised often.

**Comment by markphillips100** [comment]
Hi @negz , yes I did try the script first, using gitbash on windows but I think I may have then downloaded the exe and dropped in a path where it's being executed instead of what the script provides.

I've removed all previously downloaded binaries from wherever they were moved to, re-ran script, and moved file to a location referenced by PATH.  It now executes from (my regular) powershell terminals fine and shows the beta commands.

Thanks for the help, and consider the issue closed from my side.
